docs/reference/commandline/cli.md

```diff
@@ -78,6 +78,11 @@ For example:
 Instructs Docker to use the configuration files in your `~/testconfigs/`
 directory when running the `ps` command.
 
+> **Note:**
+> Calling docker with `sudo` results in Docker looking first in `/root/.docker/`.
+> If it isn't found it looks for the user who did the `sudo` call and resolves the
```

Comment:
I would say the opposite :/ sudo something as my runcom user will be executed with root :/ btw, I don't have a strong opinion :)